### PR TITLE
Added general colors to the tailwind config

### PR DESCRIPTION
**Pull request by esther921**
# Description

Added general colors to th etailwind config

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -6,10 +6,15 @@ module.exports = {
   ],
   theme: {
     extend: {
+      colors: {
+        purple: "#5602E0",
+        "dark-gray": "#9A9A9A",
+        "light-gray": "#E4E4E4",
```
Comment (NNesta) [inline tailwind.config.js]: It could be better if you write this color in consistent way like all in quotes or no quotes at all

**Review by NNesta — CHANGES_REQUESTED** [review]
Good job

**Review by NNesta — CHANGES_REQUESTED** [review]
Good job

File: tailwind.config.js (hunk 1 above)
Comment (esther921) [reply tailwind.config.js]: I tried it but then when I save they automatically get removed, to mean it is a correction itself.

**Review by NNesta — APPROVED** [review]
Well done

**Review by Fidesnoella — APPROVED** [review]

**Pull request merged**